main.css

```diff
@@ -0,0 +1,126 @@
+@import url("style-guide/compiled-style-guide.css");
+body {
+    background-color: var(--bg-color);
+    font-family: 'Exo', sans-serif;
+    color: var(--text-color);
+}
+
+
+/* NAVBAR CSS */
+
+.navbar {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--primary-bg);
+    border-bottom-left-radius: 0.7rem;
+    border-bottom-right-radius: 0.7rem;
+    box-shadow: var(--default-box-shadow-format);
+    color: var(--primary-color);
+}
+
+.nav-links {
+    display: flex;
+    justify-content: space-between;
+    gap: 0.7rem;
+}
+
+.nav-links .link-none:hover {
+    color: var(--primary-color-hover);
+}
+
+
+/* MAIN HERO SECTION CSS */
+
+.hero {
+    display: grid;
+    grid-template-columns: 50% 50%;
+}
+
+.hero-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.hero-highlight {
+    background: #c21500;
+    background: -webkit-linear-gradient(to top, var(--primary-color-hover), var(--secondary-color-hover));
+    background: linear-gradient(to top, var(--primary-color-hover), var(--secondary-color));
+    background-clip: text;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+
+.hero-button {
+    margin-top: 8%;
+    background-color: var(--primary-bg);
+    color: var(--text-color);
+    border: 0px;
+    border-radius: 8px;
```

Comment:
Use rems or % for responsiveness, Mostly you used them everywhere, missed it here.